hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/offlineImageViewer/OfflineImageViewerPB.java

```diff
@@ -132,6 +133,7 @@ private static Options buildOptions() {
     options.addOption("delimiter", true, "");
     options.addOption("sp", false, "");
     options.addOption("t", "temp", true, "");
+    options.addOption("threads", true, "");
```

Comment:
I am concerned that parameter `-threads` will collide with `-t`, and it could be parsed to `-t hreads` here. It is safe to changes names to another one to avoid abuse.

Reply:
Yes, it should be renamed and will be fixed in next commit. By the way, is there a suitable parameter name recommendation ? 😊

Reply:
This option name `threads` has to rename here. Otherwise it will conflict with `-t`.

Reply:
> This option name `threads` has to rename here. Otherwise it will conflict with `-t`.

New option as follow. Point it out If any better option, thanks!
```java
options.addOption("m", "multiThread", true, "");
```